doc/source/testingfeatures.rst

```diff
@@ -0,0 +1,80 @@
+====================
+Testing new features
+====================
+
+New features are constantly being developed for SunPy by contributors from
+around the world. These new features are normally submitted as a
+:ref:`Pull Request <dev-reference-label>`. Due to the review process imposed
+on pull requests to maintain the stability and consistency of the code base,
+it may take some time for a new feature to get accepted into the `master`
+branch, and a bit more to get into the next stable release version.
+
+In order to make it easier for people to test out new features (and hopefully provide valuable
+feedback to the authors) we've put together this tutorial.
+
+This assumes you have installed sunpy using anaconda, though a similar result
+can be obtained using `virtualenv`.
+
+We start by creating a new environment, so your current one still works as
+expected::
+
+ conda create -n sunpy-featureX python=3.5 sunpy
```

Comment:
I recommend setting this to `python=3` because we should default to 3.x not 2.7.